docs-gen/content/rule_set/properties.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Property Entry"
+date: 2023-11-07
+weight: 1
+---
+
```

Comment:
The term property is already used to define "members" of a struct, see https://github.com/COVESA/vehicle_signal_specification/blob/master/docs-gen/content/rule_set/data_entry/data_types_struct.md

We may theoretically live with that if define two types of properties in different context (struct property in type file, data property somewhere else) but there is a risk for confusion so changing name on one of them likely makes sense. An easy approach would be "dataproperty" and "structproperty", but there is possibly better names. As part of this we need to discuss how/where properties should be defined. In any of the existing trees (signal tree and type tree) or in a new tree? We should also state some guidelines/decisions on how the VSS-project intends to work with (data)properties. Do we intend to add data properties to the standard catalog (short term or long term), or is it totally up to user?

Reply:
Name: We can get rid of Property altogether and just promote DataProperty and ObjectProperty to the type.  (Unfortunately, "Attribute" was also used.)

Supposing this PR gets merged, it would be rapidly followed by a series of PR's that contain properties and add a reference to them in the signals.  They are definitely meant to be part of the catalog and not up to the user.  I would not add them to the existing tree as they are meant to be reused.